scripts/build-image.sh

```diff
@@ -34,9 +34,9 @@ DPKG_DEPENDS="bzr \
               postgresql-client \
               postgresql-common \
               python \
-              python-setuptools"
-DPKG_UNNECESSARY="libpython3.4 \
-                  libpython3.4-minimal"
+              python-setuptools \
+              python3"
```

Comment:
Is not the default python version for ubuntu16?

Reply:
@moylop260 not for these images, the default version for 16.04 is 2.7 that's why we install it directly [here](https://github.com/Vauxoo/docker-ubuntu-base/blob/16.04/scripts/build-image.sh#L36)